common/config/rush/version-policies.json

```diff
@@ -18,5 +18,10 @@
     "policyName": "property-grid",
     "definitionName": "individualVersion",
     "lockedMajor": 1
+  },
+  {
+    "policyName": "imodel-content-tree",
+    "definitionName": "individualVersion",
+    "lockedMajor": 0

```

Comment:
nbd, but we're going to version this as 0.x? will this be a beta pkg?

Reply:
We're still not 100% certain how the hierarchy should look like. @Carl-Hinkle said he'd like DR to consume this and see the results and go from there. So yes, this is still beta - we're likely to change the hierarchy that the component shows.